android/modules/ui/src/java/ti/modules/titanium/ui/widget/tabgroup/TiUITabLayoutTabGroup.java

```diff
@@ -90,6 +90,17 @@ protected void onLayout(boolean hasChanged, int left, int top, int right, int bo
 				super.onLayout(hasChanged, left, top, right, bottom);
 				insetsProvider.setTopBasedOn(this);
 			}
+
+			@Override
+			protected void onConfigurationChanged(Configuration newConfig)
+			{
+				super.onConfigurationChanged(newConfig);
+				if (newConfig.orientation == Configuration.ORIENTATION_PORTRAIT) {
+					mTabLayout.setTabGravity(TabLayout.GRAVITY_FILL);
+				} else if (newConfig.orientation == Configuration.ORIENTATION_LANDSCAPE) {
+					mTabLayout.setTabGravity(TabLayout.GRAVITY_CENTER);
+				}
+			}
```

Comment:
Minor nit-pick. You can get rid of the `mTabLayout.` part since you're overriding the `TabLayout` class' `onConfigurationChanged()` method. The rest is fine. Nice and simple solution.